newIDE/app/src/Utils/Haptic.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+import { isNativeMobileApp } from './Platform';
+
+export const hapticFeedback: ?({
+  durationInMs: number,
+}) => void = !isNativeMobileApp()
+  ? ({ durationInMs }) => {
+      if (window.navigator && window.navigator.vibrate) {
+        window.navigator.vibrate(durationInMs);
```

Comment:
should this be try catched just for security? unsure

Reply:
For the sake of avoiding a crash for such a useless reason, I added it